src/utils/error-utils.js

```diff
@@ -15,10 +15,22 @@ import { StatusCodes } from 'http-status-codes'
  */
 
 export const failAction = (_request, h, err) => {
-  const errors = err.details.map((detail) => ({
-    message: detail.message,
-    property: detail.path[0],
-    value: err._original[detail.path[0]]
-  }))
-  return h.response({ errors }).code(StatusCodes.BAD_REQUEST).takeover()
+  const formattedErrors =
+    err?.details?.map((detail) => {
+      const detailPath =
+        detail.path && detail.path.length > 0 ? detail.path[0] : undefined
```

Comment:
For simplicity's sake, this might be better rendered as `detail?.path?.length > 0 ? detail.path[0] : undefined`